docs/content/preview/yugabyte-cloud/cloud-overview.md

```diff
@@ -12,149 +11,166 @@ menu:
+        <div class="other-content">
+          <div class="heading"><a href="../cloud-clusters/" title="Manage">Manage</a></div>
+          <div class="detail-copy">Scale clusters, configure backups, set maintenance windows, and pause or delete clusters.</div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="three-box-row">
+  <div class="row">
+    <h2 class="col-12">More resources</h2>
+    <div class="col-12 col-md-6 col-lg-4">
+      <div class="box border">
+        <div class="other-content">
+          <div class="heading">Manage your account</div>
+          <ul>
+            <li><a class="text-link" href="../cloud-admin/manage-access/" title="Add Users">Add Users to your account</a></li>
+            <li><a class="text-link" href="../cloud-admin/cloud-billing-profile/" title="Manage Billing">Manage billing</a></li>
+            <li><a class="text-link" target="_blank" href="https://yugabyte.stoplight.io/docs/managed-apis" title="Manage Billing">YugabyteDB Managed REST API</a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div class="col-12 col-md-6 col-lg-4">
+      <div class="box border">
+        <div class="other-content">
+          <div class="heading">More Information</div>
+          <ul>
+            <li><a class="text-link" href="../release-notes/" title="Whats new">What's new</a></li>
+            <li><a class="text-link" href="../cloud-troubleshoot/" title="Troubleshoot">Troubleshoot</a></li>
+            <li><a class="text-link" href="../../faq/yugabytedb-managed-faq/" title="FAQ">FAQ</a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div class="col-12 col-md-6 col-lg-4">
+      <div class="box border">
+        <div class="other-content">
+          <div class="heading">Support</div>
+          <ul>
+            <li><a class="text-link" target="_blank" href="https://support.yugabyte.com/hc/en-us/requests/new?ticket_form_id=360003113431" title="Contact Support">Contact Support</a></li>
+            <li><a class="text-link" target="_blank" href="https://status.yugabyte.cloud/" title="YugabyteDB Managed Status">YugabyteDB Managed Status</a></li>
+            <li><a class="text-link" target="_blank" href="https://communityinviter.com/apps/yugabyte-db/register" title="Join Our Community">Join Our Community</a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="three-box-row">
+  <div class="row">
+    <h2 class="col-12">Continue learning</h2>
+    <div class="col-12 col-md-6 col-lg-4">
+      <div class="box border">
+        <div class="other-content">
+          <div class="heading">Build Applications</div>
+          <div class="tag-line">FROM EXAMPLES</div>
+          <ul>
+          <div class="detail-copy">Start coding in your favorite programming language using these examples.</div>
+          <a class="text-link" target="_blank" href="../develop/build-apps/" title="Get Started">Get Started</a>
```

Comment:
This link is returning a 404. The PR does not have the latest changes thats under the _Develop_ section.